### ISSUE TITLE
How do i set default error level to warn ? 

### ISSUE THREAD

**Issue by theapache64** [eslint/eslint#10438]
It seems that default error level for all eslint rules are "error". This is annoying as my app doesn't compile even for an omitted semicolon.

How can I set it to "warn" so that my app compiles but shows warnings?

I know I can set each rule to warn manually but I'd prefer to do it globally. In the official docs I haven't found such option.

**Comment by nzakas** [comment]
The best option for this to to use a JavaScript configuration file (`.eslintrc.js`) and assign a variable for the error level that you then pass to each rule, such as:

```js
const severity = "warn";

module.exports = {
    rules: {
        semi: severity,
        indent: severity
        // and so on
    };
};
```

As this is a question rather than an action item, I'm closing the issue. If you still need help, please send a message to our [mailing list](https://groups.google.com/group/eslint) or [chatroom](https://gitter.im/eslint/eslint). Thanks!

**Comment by slikts** [comment]
That's unfortunate, and the suggested solution to override severity manually is not practicable for large rulesets. tslint has an option to set the default severity. The issue it fixes is that a lot of the "errors" are just autofixed on the next save, so it's just an irritation to see them. For example, I would change something, and it'd light it all up making me think there's a syntax error, when in reality it's just the line length.
